### DISCUSSION TITLE
OP10 Pro NE2215

### DISCUSSION THREAD

**Discussion by wrekked1975** [phhusson/treble_experimentations#2334] [General]
I have been flashing GSI for a year now and I recently purchased this OnePlus 10 Pro Unlocked Ver NE2215 , It came stock Android 12, I tried the conventional ways to flash the thing and I cannot figure out how to change the file system so that I can mount the system or create partitions as it is a Super Dynamic Seamless thing and I am at my last nerve with it. There is no TWRP, no custom recovery, ROM or support nor MSM tool. I can't even get it back to stock as it only lets me boot to bootloader Fastbootd and the stock recovery which is a mystery to me. Any help would be greatly appreciated.

**Comment by devrieshh** [comment]
cant you just flash in fastbootd with the command ./fastboot flash system system-squeak-arm64-ab-vanilla.img
